CSS Assignment 1/profile.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+	<!--Header starts-->
+	<head>
+		<title>Profile Page</title>
+		<meta name = "author" content = "Akshita Dixit">
+	<link rel="stylesheet" href="css/style.css">
+	</head>
+	<!--Main Body starts-->
+	<body>
+		<!--Navigation bar starts-->
+		<nav style="margin-top : -40px; margin-left : -200px ; margin-right : -200px; float : right;">

```

Comment:
When using CSS, all style information should be in the CSS files, and none in the HTML file.